x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_embeddable_chart/timeseriesexplorer_title.tsx

```diff
@@ -0,0 +1,152 @@
+  return (
+    <EuiToolTip
+      content={
+        isAddFilter ? (
+          <FormattedMessage
+            id="xpack.ml.timeSeriesExplorer.title.addFilterTooltip"
+            defaultMessage="Add filter for value"
+          />
+        ) : (
+          <FormattedMessage
+            id="xpack.ml.timeSeriesExplorer.title.addNegateFilterTooltip"
+            defaultMessage="Add negate filter for value"
+          />
+        )
+      }
+    >
+      <EuiButtonIcon
+        size="xs"
+        onClick={() => {
+          const trigger = uiActions.getTrigger(SINGLE_METRIC_VIEWER_ENTITY_FIELD_SELECTION_TRIGGER);
+          trigger.exec({
+            embeddable: api,
+            data: [
+              {
+                ...entity,
+                operation: isAddFilter
+                  ? ML_ENTITY_FIELD_OPERATIONS.ADD
+                  : ML_ENTITY_FIELD_OPERATIONS.REMOVE,
+              },
+            ],
+          });
+        }}
+        iconType={isAddFilter ? 'plusInCircle' : 'minusInCircle'}
+        aria-label={
+          isAddFilter
+            ? i18n.translate('xpack.ml.timeSeriesExplorer.title.addFilterAriaLabel', {
+                defaultMessage: 'Add filter',
+              })
+            : i18n.translate('xpack.ml.timeSeriesExplorer.title.addNegateFilterAriaLabel', {
+                defaultMessage: 'Add negate filter',
+              })
+        }
+      />
+    </EuiToolTip>
+  );
+};
+
+interface SingleMetricViewerTitleProps {
+  api: any;
+  entityData: { entities: MlEntity[]; count: number };
+  functionLabel: string;
+}
+
+export const SingleMetricViewerTitle: FC<SingleMetricViewerTitleProps> = ({
+  api,
+  entityData,
+  functionLabel,
+}) => {
+  return (
+    <EuiFlexGroup direction="column" gutterSize="none">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup gutterSize="none" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiTitle size={'xs'}>
+              <h2>
+                <span>
+                  {i18n.translate('xpack.ml.timeSeriesExplorer.singleTimeSeriesAnalysisTitle', {
+                    defaultMessage: 'Single time series analysis of {functionLabel}',
+                    values: { functionLabel },
+                  })}
+                </span>
+              </h2>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <TimeSeriesExplorerHelpPopover embeddableMode />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup alignItems="center">
+          {entityData.entities.map((entity, i) => {
+            return (
+              <EuiFlexItem grow={false}>
+                <EuiFlexGroup gutterSize="none" alignItems="center">
+                  <EuiFlexItem grow={false}>
+                    <EuiTextColor color={'success'} component={'span'}>
+                      {`(${entity.fieldName}: ${entity.fieldValue})`}
```

Comment:
We have different styling for the partition fields between the anomaly charts and the SMV embeddable - using badges in the anomaly charts, and text here. What do you think about using the badge for both, or switching the charts to use the same visual as here?

<img width="1208" alt="Screenshot 2024-11-06 at 11 35 01" src="https://github.com/user-attachments/assets/5c15c8b0-400b-4af6-a8ab-0f9d518fbcf5">

Reply:
Updated in 2400d6127495bc3a5390abb1a804a787a2acea8e

Reply:
Do we need the parentheses around the entity field name and value? Not sure they add anything.

Reply:
🤔 I think just for readability? Happy to remove them for both the smv and charts if they aren't necessary.

Reply:
Removed parentheses: 
<img width="1723" alt="image" src="https://github.com/user-attachments/assets/d5d113bf-fe11-4503-ac85-31f0e32fe170">